ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Ledger/Config.hs

```diff
@@ -9,12 +9,13 @@
 {-# OPTIONS_GHC -Wno-orphans #-}
 module Ouroboros.Consensus.Shelley.Ledger.Config (
     BlockConfig (..)
-  , mkShelleyBlockConfig
   , CodecConfig (..)
+  , CompactGenesis
   , StorageConfig (..)
-  , CompactGenesis -- opaque
```

Comment:
And here's one of the `-- opaque` comments I had expected.

Reply:
Yeah, this is not great. As you mentioned, we could change to haddock and see how that goes.